files/velero/values.yaml

```diff
@@ -14,20 +18,19 @@ serviceAccount:
       eks.amazonaws.com/role-arn: ${role_arn}
 
 initContainers:
-- name: velero-plugin-for-aws
-  image: velero/velero-plugin-for-aws:${aws_container_version}
-  imagePullPolicy: IfNotPresent
-  volumeMounts:
-    - mountPath: /target
-      name: plugins
+  - name: velero-plugin-for-aws
+    image: velero/velero-plugin-for-aws:${aws_container_version}
+    imagePullPolicy: IfNotPresent
+    volumeMounts:
+      - mountPath: /target
+        name: plugins
 
 schedules:
   fullbackup:
     disabled: false
-    schedule: "0 * * * *"
+    schedule: "0 4 * * *"
```

Comment:
I would add this configuration as variable. What do you think?

Reply:
Done